### PR TITLE
Removing unused imports and adding a log parameter for placeholder

### PR DESCRIPTION
**Pull request by sarveshkaushal**
I have added 2 changes:
1. Removed unused imports
2. Added missing log parameter for placeholder

### CODE REVIEW

File: util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java [hunk 1]
```diff
@@ -125,6 +125,7 @@ public void addEventHandlerWithResyncPeriod(
           log.warn(
               "DefaultSharedIndexInformer#resyncPeriod {} is smaller than resyncCheckPeriod {} and the informer has already started. Changing it to {}",
               resyncPeriodMillis,
+              resyncCheckPeriodMillis,
```
Comment (brendandburns) [inline util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java:128]: why is this change needed?

Comment (sarveshkaushal) [reply util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java:128]: If you look at the log message- _"DefaultSharedIndexInformer#resyncPeriod {} is smaller than resyncCheckPeriod {} and the informer has already started. Changing it to {}"_. It has 3 placeholders but we are supplying only 2 variables to log.

Comment (sarveshkaushal) [reply util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java:128]: I can remove this if this looks redundant.

Comment (brendandburns) [reply util/src/main/java/io/kubernetes/client/informer/impl/DefaultSharedIndexInformer.java:128]: No that's fine, I was just confused about how adding a parameter was necessary if it was compiling before, but now I realize that it's a varargs function.
